src/Bicep.Core/Syntax/SyntaxFacts.cs

```diff
@@ -69,5 +69,32 @@ TokenType.MultilineString or
             TokenType.AsKeyword => "as",
             _ => null,
         };
+
+        public static CommentStickiness GetCommentStickiness(TokenType type) => type switch
```

Comment:
>GetCommentStickiness [](http://example.com/codeflow?start=40&length=20)

Is it worth documenting why certain token types have specific comment stickiness?